tests/System/integration/site/components/com_contact/Form.cy.js

```diff
@@ -0,0 +1,11 @@
+describe('Test that the form view ', () => {
```

Comment:
```suggestion
describe('Test that the form view ', () => {
  afterEach(() => cy.task('queryDB', 'DELETE FROM #__contact_details'));

```
As you are creating a contact outside of the test control, the contacts table must be erased after each test run. Otherwise you get errors when you run the test a second time.